.gitignore

```diff
@@ -22,3 +22,6 @@ dist-ssr
 *.njsproj
 *.sln
 *.sw?
+
+# Database
+db.json
```

Comment:
@eXConf I wouldn't ignore it, so upon 'git clone' a developer would have something to look at